### PR TITLE
DNA for Perenolde, Menethil, Fordragon, Trollbane + Others

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Made very small fixes to **Anasterian**, **Lor'themar**, and **Alleria**. _(These are barely noticeable so they are not included in the list.)_
- Added **15** new character portraits:

_[Link to DNA spreadsheet](https://docs.google.com/spreadsheets/d/1Cf0FGh_iNoyDsIwceDkL_Fr4B7_rOZKaa0QbzLNqmYQ/edit#gid=652876950) for additional information, such as what was used as reference for the portraits._

### Humans

**Perenolde**
- Aliden Perenolde (by @ercarp)
- Beve Perenolde (by @ercarp)
- Isolde Perenolde (by @ercarp)
- Isiden Perenolde (by @ercarp)
- **_Non-Canon_** Valerius Perenolde (by @ercarp)
- **_Non-Canon_** Malatar Perenolde (by @ercarp)

<details><summary>Screenshots</summary>

image
image
image
image
image
image

</details>

**Menethil**
- Lianne Menethil (by @ercarp)
- Calia Menethil (by @ercarp)

<details><summary>Screenshots</summary>

image
image

</details>

**Fordragon**
- Bolvar Fordragon (by @ercarp)
- Taelia Fordragon (by @ercarp)

<details><summary>Screenshots</summary>

image
image

**Crispy Bolvar** _(Concept)_
image

</details>

**Trollbane**
- Danath Trollbane (by @ercarp)
- Galen Trollbane (by @ercarp)

<details><summary>Screenshots</summary>

image
image

</details>

**Blackmoore**
- Aedelas Blackmoore (by @ercarp)

<details><summary>Screenshot</summary>

image

</details>

**Dathrohan**
- Saidan Dathrohan (by @ercarp)

<details><summary>Screenshots</summary>

image

</details>

**Shaw**
- Mathias Shaw (by @ercarp)

<details><summary>Screenshots</summary>

image

He will use a custom beard made by @zumbak04:

image
image

</details>

## Changes applied:
- d26af9532d6598170204fc4599eb16758c05cf29: Made Saidan Dathrohan's skin more tan (by @ValianBlue)
- edc0aec48a83aada24e6df5d5b0a877718382893: Changed Bolvar's hairstyle to a more fitting one (by @ValianBlue)
- f333f6f40351322b249ac567eba28a7eec480878: Reworked Taelia's DNA to look more like Bolvar (by @ValianBlue)
- 0e4057471e995bd01492c7e3ac0dc327d7b0f039: Made Taelia's skin more tan (by @ValianBlue)
- f5d10f09ada1250cfb56d041c62cb790eaf8bb0c: Made Aedelas's cheeks rounder (by @Remerod)

# How to test:
Consider the following when giving your feedback:
- **Canonicity**: Observe if the character's appearance matches the information provided by canon in terms of hair, eye, and skin color; hairstyle, facial hair; body shape, etc.
- **Familiality**: Observe whether or not characters look sufficiently related to each other if they have family members with custom DNA.
- **General Feedback**: Give general feedback on how you feel about each addition. In simple terms: does the portrait look good to you personally?

**How to view the portraits**:

- You can either **A)** rely on the screenshots provided above (although most portraits look much better once you see them in motion) or **B)** have a look at the portraits in-game.
  - Enable [Debug Mode](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode) and use the `age [amount] [character id]` console command to change a character's age if they are too young or old in the current bookmark.

### CODE REVIEW

**Review by ValianBlue — COMMENTED** [review]
_Bolvar and Taelia don't look directly related at a glance. This could be justified by Taelia looking more like her mother. ~~I think Bolvar could use some adjustments to fit his Shadowlands appearance (in terms of face shape).~~_

_I'm curious how Saidan Dathrohan would look with a slightly tanner skin tone._

_Aedelas' scar isn't 100% accurate, but is still doable._

_I'm very fond of Calia's appearance. An eerie reminiscence of Arthas which I think is great._

_Not all hairstyles are accurate. Mathias sticks out to me the most. I think for a bandaid fix we could give him a hood and that would work._

_Side Note: Bolvar reminds me a bit of Sean Bean. Nothing wrong with this, I'm just bit fascinated by it._

<details>
<summary></summary>

image
</details>

**Comment by ercarp** [comment]
> _Bolvar and Taelia don't look directly related at a glance. This could be justified by Taelia looking more like her mother._

The art linked below was my main reference for Taelia, although I used Bolvar's DNA as the base for her portrait so there should still be some remnants of Bolvar's DNA in her portrait. But as you said, it can easily be justified by her looking more like her mother (who we don't know anything about).

<details><summary>Click to expand</summary>

image

</details>

<details><summary>Click to expand</summary>

**Bolvar DNA**
image

**Taelia DNA**
image

</details>

> _I'm curious how Saidan Dathrohan with a slightly tanner skin tone._

Something like this? Good? Bad? I'd be okay with it personally as he does look quite tan in most artwork.

<details><summary>Click to expand</summary>

image
image

</details>
 
> _Aedelas' scar isn't 100% accurate, but is still doable._

It's just reusing the scar we made for Lor'themar. I figured it was good enough since the difference is so slight.

> _I'm very fond of Calia's appearance. An eerie reminiscence of Arthas which I think is great._

Thank you. I used Arthas's new DNA to make her so they would look more like brother and sister. 😃

**Comment by ValianBlue** [comment]
> 
> 
> > _Bolvar and Taelia don't look directly related at a glance. This could be justified by Taelia looking more like her mother._
> 
> The art linked below was my main reference for Taelia, although I used Bolvar's DNA as the base for her portrait so there should still be some remnants of Bolvar's DNA in her portrait. But as you said, it can easily be justified by her looking more like her mother (who we don't know anything about).

_Ah, seeing their DNA is really helpful. Bolvar has a very defined jawline, whereas Taelia's is sharper. I would try out widening her chin / jaw just a bit more. It looks a bit larger in the artwork you posted compared to her in-game model as well (although this could just be the angle)._

> 
> > _I'm curious how Saidan Dathrohan with a slightly tanner skin tone._
> 
> Something like this? Good? Bad? I'd be okay with it personally as he does look quite tan in most artwork.

_I really dig this change. It helps him stand out too which is a plus._

> 
> > _Aedelas' scar isn't 100% accurate, but is still doable._
> 
> It's just reusing the scar we made for Lor'themar. I figured it was good enough since the difference is so slight.

_Aye, it's definitely doable._

> 
> > _I'm very fond of Calia's appearance. An eerie reminiscence of Arthas which I think is great._
> 
> Thank you. I used Arthas's new DNA to make her so they would look more like brother and sister. 😃

_I'd say it worked perfectly then. :wink:_

**Comment by ercarp** [comment]
> _Ah, seeing their DNA is really helpful. Bolvar has a very defined jawline, whereas Taelia's is sharper. I would try out widening her chin / jaw just a bit more. It looks a bit larger in the artwork you posted compared to her in-game model as well (although this could just be the angle)._

Applied these suggestions to Taelia's DNA. Let me know if she looks better now. Might not look as close to the artworks of her anymore, but she should look more like Bolvar's daughter at the very least. 😛 I left her neck skinnier by intent.

<details><summary>Click to expand</summary>

image

</details>

> _I really dig this change. It helps him stand out too which is a plus._

Great! In that case I'll apply the new skin color to his DNA.

**Comment by ValianBlue** [comment]
_I really like Taelia's updated appearance. I looked at her in-game model more closely, and I'm not sure if it's me but she does seem to have tanner skin as well. Maybe this is from her mother? Also, do you have a close-up pictures of her eyes? I can't quite tell which color they are in CK3._

_Side Note: I am really fond of the artwork you used for Taelia, but I don't think it's official art. No harm in looking at other artists' perspectives though, of course. 🙂_

**Comment by ercarp** [comment]
> 
> 
> _I really like Taelia's updated appearance. I looked at her in-game model more closely, and I'm not sure if it's me but she does seem to have tanner skin as well. Maybe this is from her mother? Also, do you have a close-up pictures of her eyes? I can't quite tell which color they are in CK3._

Her eyes are dark green/hazel and her skin is _supposed_ to be tan but it doesn't seem to show that well in-game. I used her WoW model as a color reference.

<details><summary>Click to expand</summary>

image

</details>

<details><summary>Click to expand</summary>

image

</details>

> _Side Note: I am really fond of the artwork you used for Taelia, but I don't think it's official art. No harm in looking at other artists' perspectives though, of course. 🙂_

That's only because there was unfortunately very little official art. 😕

This is the only official art for Taelia (which I did have loaded up in my reference sheet as well when I made her portrait):

<details><summary>Click to expand</summary>

image

</summary>

**Comment by ercarp** [comment]
@ValianBlue Taelia's skin is now more tan. Should be closer to her skin color in WoW.

<details><summary>Click to expand</summary>

image

</details>

**Review by ValianBlue — APPROVED** [review]
_Looks perfect. 🙂_

_I approve these changes, and hopefully a few more testers can give each portrait a good look as well in case I missed some things._

**Review by Remerod — COMMENTED** [review]
I think almost every character is is very well thought of and well done! they are look very good, i have a nit pick maybe on **Aedelas Blackmoore** but other then that, they look fantastic. Good Art!
<details>
<summary>Click to expand</summary>

I think Adelas looks really good, the only thing that i can point out, is that his cheek's might look a bit long, while i think they would make him look like himself more like if they where rounder? but i dunno.
image
image
image
image
But i dunno, i might be talking gibberish. But i think his cheek's are a part of his sneakingly scheming face :P

</details>

**Comment by ercarp** [comment]
> I think Adelas looks really good, the only thing that i can point out, is that his cheek's might look a bit long, while i think they would make him look like himself more like if they where rounder? but i dunno.

I changed his `cheek_fat` gene and made his cheeks wider and slightly lower:

<details><summary>Click to expand</summary>

image
image

</details>

For the record, this was my main reference for Aedelas and in it, his cheeks are shown to be more narrow, but I think the rounder, lower cheeks make him look a bit more unique which is always a good thing:

<details><summary>Click to expand</summary>

image

</details>

**Review by Remerod — APPROVED** [review]
LGTM now!

File: common/dna_data/wc_human_dna.txt [hunk 1]
```diff
@@ -118,1536 +143,1484 @@ daelin_proudmoore_dna = { # by ercarp
...
+			tail={ "no_tail" 0 "no_tail" 0 }
+			hooves={ "no_hoves" 0 "no_hoves" 0 }
+		}		
 	}
 	enabled=yes
 }
 
-nathanos_marris_dna = { #by ercarp
+# Derek Proudmoore
+derek_proudmoore_dna = { # by icelille999
```
[the remark this answers is not in the document]
Comment (ercarp) [reply common/dna_data/wc_human_dna.txt]: The changes to Derek's DNA are meant to be on `dna-4`, that's why he's not mentioned anywhere here.

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Pull request merged**